src/views/routes/RegisterView.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react'
+import RegisterForm from '../../components/RegisterForm'
+import { User } from '../../entities/User'
+import api from '../../networking/api'
+import { useAlert } from '../shared/AlertProvider'
+
+const RegisterView = () => {
+    const [user, setUser] = useState<User.Request>(User.empty())
+    const { success } = useAlert()
+
+    const signupUser = () => {
+        api.user.signup(user).then(() => {
+            // print success message
```

Comment:
I would remove this comment, is no longer needed

Reply:
removed comment